lib/bill_forward/entities/account.rb

```diff
@@ -7,6 +7,22 @@ module BillForward
   class Account < MutableEntity
   	@resource_path = BillForward::ResourcePath.new("accounts", "account")
 
+    class << self
+      def credit(id, query_object = {}, custom_client = nil)
```

Comment:
Although, actually there is one more factor here.

```ruby
acc.credit
```

If (one day) we add a field to `Account`, called `credit`… then this method cause a name clash.

So… maybe that is a good reason not to use methods so much on billing entities in the Ruby SDK.